### PR TITLE
Min/Max loc unit test

### PR DESCRIPTION
**Pull request by artv3**
Unit test to make sure that Min/Maxloc returns the location of the first min/max value.

### CODE REVIEW

File: include/RAJA/util/Operators.hpp [hunk 1]
```diff
@@ -427,7 +427,7 @@ struct minimum : public detail::binary_function<Arg1, Arg2, Ret>,
   RAJA_HOST_DEVICE constexpr Ret operator()(const Arg1& lhs,
                                             const Arg2& rhs) const
   {
-    return (lhs < rhs) ? lhs : rhs;
+    return (rhs < lhs) ? rhs : lhs;
```
Comment (davidbeckingsale) [inline include/RAJA/util/Operators.hpp]: Do you think this change should have a comment with it?

Comment (artv3) [reply include/RAJA/util/Operators.hpp]: Sure, I can add that in.

File: include/RAJA/util/Operators.hpp [hunk 2]
```diff
@@ -419,15 +419,20 @@ struct bit_xor : public detail::binary_function<Arg1, Arg2, Ret> {
...
 
-// comparison
-
+// comparison 
+/*!
+ Checks if the candidate (rhs) value is strictly less than
+ the current value (lhs); if so the candidate is returned. 
+ When this operator is used to cycle through an array 
+ this ensures that the location of the first min/max is kept.
+*/
```
Comment (artv3) [inline include/RAJA/util/Operators.hpp]: @davidbeckingsale , I added the comment under the comparison comment.

Comment (davidbeckingsale) [reply include/RAJA/util/Operators.hpp]: Looks good, thanks!

**Pull request merged**